packages/core/content-type-builder/admin/src/components/ContentTypeBuilderNav/index.js

```diff
@@ -36,7 +36,10 @@ const ContentTypeBuilderNav = () => {
           id: `${getTrad('plugin.name')}`,
           defaultMessage: 'Content-Types Builder',
         })}
-        searchLabel="Search..."
+        searchLabel={formatMessage({
+          id: 'content-manager.components.LeftMenu.Search.label',
+          defaultMessage: 'Search for a content type',
```

Comment:
Hello, this is a translation id that already exists. its location: packages/core/admin/admin/src/translations/en.json line: 488

Reply:
Hello, this is a translation id that already exists. its location: packages/core/admin/admin/src/translations/en.json line: 488

Reply:
Since this is a translation in another plugin, the function method is not directly generated.
My idea is that some of the same translation text is defined in one place as much as possible, so that the maintenance cost of the translation text can be reduced.
Suppose we have 100 plugins, and all 100 plugins have the same translation text, if this translation text changes, then we need to modify it 100 times.
So my suggestion is to make the copy independent of the plugin system. Or have a dedicated translation copy plugin to avoid duplicate changes in all plugins. (This is just my suggestion :))

Reply:
hi @yangfei4913438 
this sounds good indeed!
after asking my team, the idea of having separated files came from having removable plugins, so we can have a separation of concerns + avoid loading unnecessary translations if the plugin is not used!
but for generic translation like "Search" that is already used in the admin, I think it makes sense to use them 😊
I added a comment on your other PR, I think we should use the same key for both fixes, do all fixes in #13536 and close this PR!

Reply:
Changes to PR #13536 have been merged into the current PR.

Reply:
thank you @yangfei4913438  for handling it, looks good!
could you update this one too with the new global traduction you created? 🙂
I will merge your PR after this last fix 🙏

```suggestion
          searchLabel={formatMessage({
               id: 'global.search,
               defaultMessage: 'Search',
          })}
```